python/cudf/cudf/tests/test_reshape.py

```diff
@@ -529,3 +529,95 @@ def test_pivot_duplicate_error():
         gdf.pivot(index="a", columns="b")
     with pytest.raises(ValueError):
         gdf.pivot(index="b", columns="a")
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "A": ["one", "one", "two", "three"] * 6,
+            "B": ["A", "B", "C"] * 8,
+            "C": ["foo", "foo", "foo", "bar", "bar", "bar"] * 4,
+            "D": np.random.randn(24),
+            "E": np.random.randn(24),
+        }
+    ],
+)
+@pytest.mark.parametrize(
+    "aggfunc", ["mean", "count", {"D": "sum", "E": "count"}]
+)
+@pytest.mark.parametrize("fill_value", [0])
+def test_pivot_table_simple(data, aggfunc, fill_value):
+    pdf = pd.DataFrame(data)
+    ref = pd.pivot_table(
+        pdf,
+        values=["D", "E"],
+        index=["A", "B"],
+        columns=["C"],
+        aggfunc=aggfunc,
+        fill_value=fill_value,
+    )
+    cdf = cudf.DataFrame(data)
+    res = cudf.pivot_table(
+        cdf,
+        values=["D", "E"],
+        index=["A", "B"],
+        columns=["C"],
+        aggfunc=aggfunc,
+        fill_value=fill_value,
+    )
+    assert_eq(ref, res, check_dtype=False)
+
+
+def test_crosstab_simple():
+    a = np.array(
+        [
+            "foo",
+            "foo",
+            "foo",
+            "foo",
+            "bar",
+            "bar",
+            "bar",
+            "bar",
+            "foo",
+            "foo",
+            "foo",
+        ],
+        dtype=object,
+    )
+    b = np.array(
+        [
+            "one",
+            "one",
+            "one",
+            "two",
+            "one",
+            "one",
+            "one",
+            "two",
+            "two",
+            "two",
+            "one",
+        ],
+        dtype=object,
+    )
+    c = np.array(
+        [
+            "dull",
+            "dull",
+            "shiny",
+            "dull",
+            "dull",
+            "shiny",
+            "shiny",
+            "dull",
+            "shiny",
+            "shiny",
+            "shiny",
+        ],
+        dtype=object,
+    )
+    ref = pd.crosstab(a, [b, c], rownames=["a"], colnames=["b", "c"])
+    res = cudf.crosstab(a, [b, c], rownames=["a"], colnames=["b", "c"])
+    assert_eq(ref, res, check_dtype=False)
```

Comment:
We usually prefer the names `expected, actual`. Please apply to other tests, too.
```suggestion
    expected = pd.crosstab(a, [b, c], rownames=["a"], colnames=["b", "c"])
    actual = cudf.crosstab(a, [b, c], rownames=["a"], colnames=["b", "c"])
    assert_eq(expected, actual, check_dtype=False)
```

Reply:
Done.